docs/source/conf.py

```diff
@@ -29,7 +29,7 @@
 
 builtins.__LIGHTNING_SETUP__ = True
 
-IS_READTHEDOCS_BUILD = os.environ.get('READTHEDOCS', False)
+SPHINX_MOCK_REQUIREMENTS = os.environ.get('SPHINX_MOCK_REQUIREMENTS', True)
```

Comment:
setting it to true by default will also mock locally when building or testing docs. are you sure we want this?

Reply:
btw the test fails because torch is getting mocked

Reply:
resolved as cast to int so CircleCI env var << 0 is recognised :]